src/main/java/unicash/MainApp.java

```diff
@@ -171,13 +171,17 @@ protected UserPrefs initPrefs(UserPrefsStorage storage) {
 
     @Override
     public void start(Stage primaryStage) {
-        logger.info("Starting AddressBook " + MainApp.VERSION);
+        logger.info("Starting UniCash " + MainApp.VERSION);
+        Font.loadFont(getClass().getResourceAsStream("/fonts/Inter-Regular.ttf"), 12);
+        // Font.loadFont(getClass().getResourceAsStream("/fonts/Inter-Medium.ttf"), 12);
+        // Font.loadFont(getClass().getResourceAsStream("/fonts/Inter-Bold.ttf"), 12);
+        // Font.loadFont(getClass().getResourceAsStream("/fonts/Inter-SemiBold.ttf"), 12);
```

Comment:
Are these necessary? Don't think we need to comment out these lines if they're not used at all

Reply:
They are commented out because we're still in the process of dealing with formatted text display in the GUI, and so the mentioned font families are still packaged within resources, commenting this code is intended as a reminder that we still have these packages in our code, they are just not being utilized right now. 

We most likely will need them soon enough, but I can remove both the commented code and the packaged `.ttf` files for this iteration if that would be preferable!

Reply:
We can remove the comments since they're not in use and keep the assets? :o That would be easier